datasets/glossis/glossis.py

```diff
@@ -117,3 +128,21 @@
 
     # Wait for all the tasks to finish
     wait_gee_tasks(taskids)
+
+
+    # This should result in flowmap tiff files
+    flowmap_task_ids = []
+    flowmap_tiffs = []
+    for current_asset in current_assets:
+        flowmap_tiff = pathlib.Path(current_asset).with_suffix('.tif').name
+        flowmap_tiffs.append(flowmap_tiff)
+        task_id = export_flowmap(current_asset, bucket)
+        flowamp_task_ids.append(task_id)
+    wait_gee_tasks(flowmap_task_ids)
+
+    # This should result in flowmap tiles in a bucket
+    flowmap_tiles_task_ids = []
```

Comment:
Were not using this variable, so could be removed.

Reply:
comments added